index.html

```diff
@@ -1410,6 +1412,15 @@ <h1>Glossary</h1>
                 <div class="note">In some cases, touchpads (like those found on a laptop) will allow the user to scroll by "dragging" on the touchpad. However, this is generally achieved
                 by the touchpad generating "fake" mouse wheel events, so this wouldn't count as a direct manipulation.</div>
                 </dd>
+            <dt><dfn>discrete and continuous properties</dfn></dt>
+                <dd><p>Discrete properties are those properties that can only have a finite set of values. In the context of pointer events, <code>pointerId</code>,
+                    <code>pointerType</code>, <code>isPrimary</code>, and the [[UI-EVENTS]] Mouse Event model properties <code>button</code>, <code>buttons</code>,
+                    <code>ctrlKey</code>, <code>shiftKey</code>, <code>altKey</code>, and <code>metaKey</code> are discrete.</p>
```

Comment:
During last PEWG meeting, I proposed mentioning `ctrlKey`, `shiftKey`, `altKey`, and `metaKey` as "discrete properties" but didn't realize it would be problematic from `PointerEvent` perspective: a `PointerEvent` (or `MouseEvent`) is fired in response to a button-change on a pointing device, and never in response to a key-change (or "button-chance on a keyboard").  If we mention these four as discrete properties, it could be interpreted as, say, "a change in `ctrlKey` fires a `PointerEvent`".

Sorry for the oversight!

Not mentioning them here solves the problem.  Or we can say these are not pointer properties but just current states of the keyboard.  Wdyt?

Reply:
as it would seem odd to not mention those properties, since they *are* discrete (and it would look more like a mistaken omission if we left them out), i've expanded the sentence where it talks about when `pointermove` may not be sent to explicitly single them out

```
<p>For performance reasons, user agents may choose not to send a <code>pointermove</code>
            event every time a <a data-lt="discrete and continuous properties">continuous property</a>
            (such as coordinates, pressure, tangential pressure, tilt, twist, or contact geometry) or a
            <a data-lt="discrete and continuous properties">discrete property</a> relating to keyboard
            modifiers (<code>ctrlKey</code>, <code>shiftKey</code>, <code>altKey</code>,
            and <code>metaKey</code> from the [[UI-EVENTS]] Mouse Event model) of a pointer
            is updated.
```

makes the sentence a bit long, but more accurate i think. and more logical to mention this there, rather than muddying up (by omission) the definition of what is and isn't discrete